source/apps/metadata.txt

```diff
@@ -0,0 +1,250 @@
+.. _app-metadata:
+
+================
+Get App Metadata
+================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. _find-atlas-project-id:
+
+Find Your Atlas Project ID
+--------------------------
```

Comment:
high level and not very important thought: i wonder if the order of the 'Find Your Atlas Project ID' and 'Find Your App ID' sections should be reversed. 

my thought would be to put 'Find Your App ID' first b/c:

1. it's more App Services specific, and these are the app services docs. 
2. i'd imagine more users would want to find app id than project id, esp more beginner users. and the power users who want the project id would def still be able to find it.

Reply:
Works for me, will swap